packages/serverpod/lib/src/server/endpoint.dart

```diff
@@ -84,7 +85,7 @@ abstract class Endpoint {
   /// Sends an event to the client represented by the [Session] object.
   Future<void> sendStreamMessage(
       StreamingSession session, SerializableEntity message) async {
-    var prefix = moduleName == null ? '' : '$moduleName.';
+    var prefix = moduleName.isNullOrEmpty ? '' : '$moduleName.';
```

Comment:
Note: Here is a case where the program actually wants to differentiate between null and empty. Here an empty string is valid value while, and if we receive null we set the default value (which right now is empty string but could potentially be something different). We also hide away the `trim()` that is applied on the isNullOrEmpty call.

Reply:
You are right and this function any how give the expected value as if the variable is null or empty, it goes to empty string only.

Reply:
It is true that we yield the same result right now. But for program control I would argue that they have a different meaning.

One being the default value of we have no module name and the other being a defined module name of empty string.